helm_deploy/hmpps-assessments-api/templates/ingress.yaml

```diff
@@ -0,0 +1,29 @@
+{{- if .Values.ingress.enabled -}}
+{{- $fullName := include "hmpps-assessments-api.fullname" . -}}
+{{- $ingressPath := .Values.ingress.path -}}
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: {{ $fullName }}
+  labels:
+    app: {{ template "hmpps-assessments-api.name" . }}
```

Comment:
With `hmpps-assessments-api` all the helpers are prefixed `api` rather than the chart name. So I think this should be `app.name` and likewise for other `template` usages